package.json

```diff
@@ -79,10 +79,10 @@
     "grunt-contrib-uglify": "^5.2.2",
     "grunt-eslint": "^24.0.0",
     "grunt-filerev": "^2.3.1",
-    "grunt-jsonlint": "^2.1.3",
+    "grunt-jsonlint": "^1.1.0",
```

Comment:
To successfully build the app, npm needs this, otherwise the app cannot be build because of some dependency conflicts. npm does this automatically when we execute `npm audit fix`.

Reply:
The configurations are improved, no more downgrades needed.